apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx

```diff
@@ -13,7 +13,7 @@ export default function OutputFeedBack({
 
 	return (
 		<span className="text-xl">
-			{resumePath ? "Successfully uploaded " + resumePath : ""}
+			{resumePath ? "Selected " + resumePath : ""}
 		</span>
```

Comment:
Suggestion: this text is even larger than the input field label. Could we make it a bit smaller and also perhaps change the text color or decoration (maybe italic or underline) of the file name so it is a little more clear to the user?